src/MJanssen/Provider/RoutingServiceProvider.php

```diff
@@ -212,8 +204,66 @@ protected function addActions(Controller $controller, $actions, $type)
      * @param $value
      * @param $type
      */
-    protected function addAction(Controller $controller, $name, $value, $type)
-    {
+    protected function addAction(Controller $controller, $name, $value, $type) {
         call_user_func_array(array($controller, $type), array($name, $value));
     }
+
+    /**
+     * Determines whether a variable is valid Closure
```

Comment:
If we need a description what the method does then our method name isn't clear enough:

```
isClosure($param)
```

Seems to me that the method name is fine as is, it's already saying what it does